data/run_lammps_example.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+TEMPERATURE=300
+BOX_SIZE=1
+
+lmp < data/lammps_input_example.lammps -v STEP 10 -v T $TEMPERATURE -v S $BOX_SIZE
```

Comment:
the "data/" is not needed since the script and the input file are in the same folder.